aiida/cmdline/commands/cmd_group.py

```diff
@@ -136,7 +136,9 @@ def group_description(group, description):
 @arguments.GROUP()
 @with_dbenv()
 def group_show(group, raw, uuid):
-    """Show information on a given group. Pass the GROUP as a parameter."""
+    """Show information on a given group.
+
+    Pass the group as a parameter."""
```

Comment:
"Pass the group .." is superfluous. It just states that it takes an argument, but that should be clear

Reply:
Agreed. It was there and I didn't want to to remove it, but I agree it's better to remove.